.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Every time you create a file, don't forget to add a empty line at the end.
It's a convention that ease the reading when we use commands like `cat`.